source/materials/MaterialsList.cc

```diff
@@ -818,6 +818,28 @@ namespace materials {
     return mat;
   }
 
+  // WLS EJ-286
```

Comment:
Is there any difference between this definition and that of polyvinyl toluene (PVT)? What the manufacturer calls EJ286 is just PVT doped with a certain WLS. That matters for its optical properties, but not for the material definition. In the geometry I'd use PVT. This probably applies as well to EJ280, Y11, etc.

Reply:
Also, Y11 and B2 are exactly the same material, so one of those should be eliminated. 
As for the others, there are differences in the density and in the composition (number of H and C atoms), therefore we should check which materials we are really using.